### PR TITLE
feat(molecule/rating): add atomIcon dependencie in rating

### PR DESCRIPTION
**Pull request by PablitoGS**
## MOLECULE/RATING
Add AtomIcon dependency

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description, Motivation and Context

if we use svg as an icon we are not adding the dependency or styles that we have for icons

### Screenshots - Animations
Captura de pantalla 2020-11-20 a las 10 34 00

### CODE REVIEW

File: components/molecule/rating/src/Icons/starFilled.js [hunk 1]
```diff
@@ -1,9 +1,18 @@
 import React from 'react'
+import PropTypes from 'prop-types'
+import AtomIcon, {ATOM_ICON_SIZES} from '@s-ui/react-atom-icon'
 
-const StarFilled = () => (
+const starFilled = (
```
Comment (andresin87) [inline components/molecule/rating/src/Icons/starFilled.js:5]: 👍

File: components/molecule/rating/src/Icons/starHalfFilled.js [hunk 2]
```diff
@@ -1,9 +1,20 @@
 import React from 'react'
+import PropTypes from 'prop-types'
+import AtomIcon, {ATOM_ICON_SIZES} from '@s-ui/react-atom-icon'
 
-const StarHalfFilled = () => (
+const starHalfFilled = (
```
Comment (andresin87) [inline components/molecule/rating/src/Icons/starHalfFilled.js:5]: 👍

File: components/molecule/rating/src/Icons/starOutline.js [hunk 3]
```diff
@@ -1,9 +1,18 @@
 import React from 'react'
+import PropTypes from 'prop-types'
+import AtomIcon, {ATOM_ICON_SIZES} from '@s-ui/react-atom-icon'
 
-const StarOutline = () => (
+const starOutline = (
```
Comment (andresin87) [inline components/molecule/rating/src/Icons/starOutline.js:5]: 👍

File: components/molecule/rating/package.json [hunk 4]
```diff
@@ -9,7 +9,8 @@
     "build:styles": "../../../node_modules/.bin/cpx './src/**/*.scss' ./lib"
   },
   "dependencies": {
-    "@s-ui/component-dependencies": "1"
+    "@s-ui/component-dependencies": "1",
+    "@s-ui/react-atom-icon": "^1.6.0"
```
Comment (andresin87) [inline components/molecule/rating/package.json]: fix the version:
`"@s-ui/react-atom-icon": "1"`

**Review by andresin87 — CHANGES_REQUESTED** [review]

**Comment by andresin87** [comment]
Initial Capitalized variables are reserved for classes or components. Nice point. When we store a rendered node element, its containing variable is not capitalized. Thanks

File: components/molecule/rating/package.json (hunk 4 above)
Comment (PablitoGS) [reply components/molecule/rating/package.json]: Fixed!

**Review by andresin87 — APPROVED** [review]

**Comment by andresin87** [comment]
Try to preserve the default small returning value in order not to make major changes

**Comment by andresin87** [comment]
Try to still return the small size by default in order not to make a major change

**Comment by PablitoGS** [comment]
> Try to still return the small size by default in order not to make a major change

@andresin87 small size is not as default here? 
```
const MoleculeRating = ({
  numStars = 5,
  value,
  size = SIZES.SMALL,
```

**Comment by andresin87** [comment]
> > Try to still return the small size by default in order not to make a major change
> 
> @andresin87 small size is not as default here?
> 
> ```
> const MoleculeRating = ({
>   numStars = 5,
>   value,
>   size = SIZES.SMALL,
> ```

Yup!

**Pull request merged**